routing/routing.xml

```diff
@@ -953,7 +1055,7 @@
 
 		</point>
 
-		<way attribute="penalty_transition">
+		<!-- <way attribute="penalty_transition">
```

Comment:
Actually, the problem of left turns is a very good point. 

Is there any possibility to add a penalty _precisely_ for that (only left turns, only on main roads)? Maybe we can make something up? That would be great. 

At the moment it is a dilemma: If the routing engine is programmed to stay away from main roads, it will sometimes do so by exiting a main road to the left. The maps by themselves normally do not give any hint: Often the left turn is just done be entering into a residential road.  https://www.openstreetmap.org/way/66514469 Since there is no "crossing" marked, or anything that could be applied with a penalty, it would be great to just be able to put a penalty on ANY left turn on main roads.